pupil_src/shared_modules/video_export_launcher.py

```diff
@@ -45,57 +44,59 @@ def verify_out_file_path(out_file_path, rec_dir):
     return out_file_path
 
 
-def avoid_overwrite(out_file_path):
-    if os.path.isfile(out_file_path):
-        # append something unique to avoid overwriting
-        out_file_path, ext = os.path.splitext(out_file_path)
-        out_file_path += str(int(time.time())) + '.mp4'
-    return out_file_path
-
-
 class Video_Export_Launcher(Analysis_Plugin_Base):
     """docstring for Video_Export_Launcher
     this plugin can export the video in a seperate process using exporter
     """
+
     icon_chr = chr(0xec09)
-    icon_font = 'pupil_icons'
+    icon_font = "pupil_icons"
 
     def __init__(self, g_pool):
         super().__init__(g_pool)
         # initialize empty menu
         self.exports = []
-        # default_path = verify_out_file_path("world_viz.mp4",rec_dir)
-        default_path = "world_viz.mp4"
-        self.rec_name = default_path
+        self.rec_name = "world.mp4"
 
     def init_ui(self):
         self.add_menu()
-        self.menu.label = 'Export Video'
+        self.menu.label = "Export Video"
         # add menu to the window
         self._update_ui()
 
     def _update_ui(self):
         del self.menu.elements[:]
-        self.menu.append(ui.Info_Text('Supply export video recording name. The export will be in the recording dir. If you give a path the export will end up there instead.'))
-        self.menu.append(ui.Text_Input('rec_name',self,label='Export name'))
-        self.menu.append(ui.Info_Text('Select your export frame range using the trim marks in the seek bar. This will affect all exporting plugins.'))
-        self.menu.append(ui.Info_Text("Press the export button or type 'e' to start the export."))
+        # self.menu.append(ui.Info_Text('The export can be found in the exports folder, under the recording directory.'))
+        # self.menu.append(ui.Info_Text('The world video file will be named as {}'.format(self.rec_name)))
+        # self.menu.append(ui.Text_Input('rec_name',self,label='Export name', setter=lambda x: None))
+        self.menu.append(
+            ui.Info_Text(
+                "Select your export frame range using the trim marks in the seek bar. This will affect all exporting plugins."
```

Comment:
This text is deprecated as well. Export range is selected in the general settings since a few versions.